### ISSUE TITLE
RPA.Salesforce query does not produce "human readable" Tables

### ISSUE THREAD

**Issue by orlof** [robocorp/rpaframework#215]
Sometimes the raw result from Salesforce query is needed (e.g. with attributes), but quite often a more human friendly output format would be preferred. So far I have done the conversion with a code similar to this:

```
import sys
from collections import OrderedDict
from RPA.Tables import Tables

def _get_values(node, data, prefix=""):
    if isinstance(node, OrderedDict):
        for k, v in node.items():
            if k != "attributes":
                _get_values(v, data, prefix + "." + k)
    else:
        data.append((sys.intern(prefix), node))

    return data

def generate_table_from_SFDC_API_query(result, start=0, limit=0):
    records = result["records"][start:] if limit == 0 else result["records"][start:start+limit]

    if len(records) == 0:
        return Tables().create_table()

    cols = [k for k, v in _get_values(records[0], [])]
    table = Tables().create_table(columns=cols)

    for row in records:
        values = _get_values(row, [])
        table.append_row(row=dict(values))

    return table
```

It could be integrated into Salesforce lib with as an output option - unless salesforce already provides option to choose from different query result formats.

**Comment by cmin764** [comment]
Let's check this first and test it with a robot to make sure it's already implemented. (then close)

**Comment by cmin764** [comment]
Fixed by #595
